providers/openfeature-provider-flagd/src/openfeature/contrib/provider/flagd/resolvers/process/custom_ops.py

```diff
@@ -69,7 +81,7 @@ def f(s1: str, s2: str) -> bool:
 
 
 def string_comp(
-    comparator: typing.Callable[[str, str], bool], data: dict, *args: JsonLogicArg
+        comparator: typing.Callable[[str, str], bool], data: dict, *args: JsonLogicArg
```

Comment:
The formatting should be consistent with the rest of the codebase which uses `ruff`. Check out `pre-commit` and the configuration [here](https://github.com/open-feature/python-sdk-contrib/blob/main/.pre-commit-config.yaml). 

You should be able to run `pre-commit install` in the root directory and it will prevent you from committing mal-formatted code. 

```suggestion
    comparator: typing.Callable[[str, str], bool], data: dict, *args: JsonLogicArg
```

Reply:
thank you, i was struggeling with getting the linting to run - should be better now